flink-rpc/flink-rpc-akka/pom.xml

```diff
@@ -259,6 +282,7 @@ under the License.
 					</execution>
 				</executions>
 				<configuration>
+					<scalaVersion>${akka.scala.version}</scalaVersion>
```

Comment:
This was necessary because the plugin kept trying to compile with either 2.11.X or 2.12.0 for some reason.